markhor_slam/launch/sick_tim.launch

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<launch>
+    <param name="robot_description" command="$(find xacro)/xacro --inorder '$(find sick_tim)/urdf/example.urdf.xacro'" />
```

Comment:
I'm not sure if this is right, but please correct me if I misunderstand. I think that the node for the LIDAR, will use the description of the robot to do some computing of the map. I'm afraid that if we are not using the right description of the robot, we might have an incorrect rendering of the map once running on the robot. 

It would be wise to investigate the reason the node use the robot's description to understand and maybe correct the present definition.